trieste/acquisition/optimizer.py

```diff
@@ -140,3 +143,44 @@ def target_func_with_vectorized_inputs(
         return tf.reshape(vectorized_points, [batch_size, -1])  # [B, D]
 
     return optimizer
+
+
+def batchify_greedy(
+    batch_size_one_optimizer: AcquisitionOptimizer[SP],
+    batch_size: int,
+) -> AcquisitionOptimizer[SP]:
+    """
+    A wrapper around our :const:`AcquisitionOptimizer`s. This class wraps a
```

Comment:
"our" implies it's not valid for user-defined optimizers, whereas it very much is

Reply:
also, this isn't a class. You've also got the same problem on the other function